tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -1 +1,2 @@
-"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
\ No newline at end of file
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"Microsoft.Azure.Commands.ApiManagement.ServiceManagement.dll","Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Commands.SetAzureApiManagementPolicy","Set-AzureRmApiManagementPolicy","0","2050","A validate set has been added for parameter 'Format' for cmdlet 'Set-AzureRmApiManagementPolicy'.","Remove the validate set from parameter 'Format'."
```

Comment:
@solankisamir can you confirm that this isn't a breaking change? If a user were to provide values outside of the values in the `ValidateSet`, would the cmdlet throw an exception?